examples/js/exporters/GLTFExporter.js

```diff
@@ -895,7 +895,7 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
-			if ( material.isShaderMaterial ) {
+			if ( material.isShaderMaterial && !material.isGLTFSpecularGlossinessMaterial ) {
```

Comment:
It looks like a warning below, `GLTFExporter: Use MeshStandardMaterial or MeshBasicMaterial for best results.`, will still be printed with spec/gloss materials. Could we make sure that doesn't appear?

Reply:
Fixed in the last commit